src/plot/xy-plot.js

```diff
@@ -46,6 +46,28 @@ const DEFAULT_MARGINS = {
   bottom: 40
 };
 
+function cleanseData(data) {
+  return data.map(series => {
+    if (!Array.isArray(series)) {
+      return series;
+    }
+    return series.map(row => ({...row, parent: null}));
```

Comment:
i also want to re-write treemap to be in XY coordinates, so itll also be dealing with that.